website/docs/config/networks.mdx

```diff
@@ -108,7 +110,7 @@ Your `network` property should now look something like this:
 ```json
 {
    "network":{
-      "Ghostnet":{
+      "ghostnet":{
          "label":"Ghostnet Protocol Testnet",
          "rpcUrl":"https://rpc.ghostnet.teztnets.xyz",
          "protocol":"PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY",
```

Comment:
We should use the Alpha protocol here so that we don't have to keep updating this.